spec/mailers/alchemy/messages_mailer_spec.rb

```diff
@@ -36,9 +36,5 @@ class ::ApplicationMailer; end
     it "reply_to should be set to senders email" do
       expect(mail.reply_to).to eq([message.email])
     end
-
-    it "mail body includes message" do
-      expect(mail.body).to match /#{message.message}/
-    end
```

Comment:
The specs was assuring the template is present. Please revert the deletion and re-add the spec.